flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/StreamOperatorChainingTest.java

```diff
@@ -324,4 +327,18 @@ public void invoke(String value) throws Exception {
 		return mockTask;
 	}
 
+	@Test
```

Comment:
It's excellent that you added a test! I think, however, that this should go into a new file `StreamOperatorChainingITCase` because it is an integration test and not a unit test (it runs a whole Flink pipeline instead of testing an isolated component).